x-pack/platform/plugins/shared/alerting/server/alerts_service/lib/is_existing_alert.ts

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
This function doesn't feel like it fits in `alerts_service`.  Maybe `alerts_client`?  And then it could use the `search()` method of the alerts client.  https://github.com/elastic/kibana/blob/8cb4c62aeca27ff4a9f59a3cd56856410bbff92c/x-pack/platform/plugins/shared/alerting/server/alerts_client/alerts_client.ts#L264-L266

Reply:
Ah, ya, you're right.  I poked around a bit more and see similar code, that I didn't see yesterday! :-)